src/components/InputNumber.js

```diff
@@ -28,43 +28,36 @@ const InputNumber = React.forwardRef(
   (
     {
       defaultValue,
-      precision,
-      separator,
-      locale,
+      isNFT,
       onValueChange,
       requirePositive,
       ...otherProps
     },
     ref
   ) => {
-
-    const decimalPlaces = useSelector((state) => state.serverInfo.decimalPlaces);
-    const decimalPrecision = precision ?? decimalPlaces;
+    if (ref !== null) {
+        // TODO: We should just convert InputNumber from a React.forwardRef to a normal React.Component,
+        //  but we may do this in a separate PR
+        throw Error('do not use ref in InputNumber, instead use onValueChange')
```

Comment:
Can you also `console.error` this to make this error easier to catch during development.

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet/pull/699/commits/07afc271fef65073accfb308cfb99d9901b7a469